types/porty/index.d.ts

```diff
@@ -0,0 +1,67 @@
+declare namespace Porty {
+    /**
+     * The host to run on
+     * @default "0.0.0.0"
+     */
+    export let HOST: string;
+    /**
+     * Minimum port
+     * @default 8000
+     */
+    export let MIN: number;
+    /**
+     * Maximum port
+     * @default 10000
+     */
+    export let MAX: number;
+
+    export interface FindOptions {
+        /** Ports to avoid */
+        avoids?: readonly number[] | undefined;
+        /**
+         * The host to run on.
+         * Defaults to {@link HOST}
+         */
+        host?: string | undefined;
+        /**
+         * Minimum port.
+         * Defaults to {@link MIN}
+         */
+        min?: number | undefined;
+        /**
+         * Maximum port.
+         * Defaults to {@link MAX}
+         */
+        max?: number | undefined;
+        /**
+         * The current port to check.
+         * This parameter is incremented while {@link find} is run,
+         * and starts at {@link min} if unspecified
+         */
+        port?: number | undefined;
```

Comment:
This is likely internally used and not meant to be accepted as a parameter AFAICT.

Reply:
I agree, but I think it's worth mentioning since it does exist on the object. Maybe I should mark it `@internal` and/or `@private` in the JSDoc?